tox.ini

```diff
@@ -28,24 +30,20 @@ deps =
 commands =
     zope-testrunner --test-path=src
 
-[testenv:py34-pure]
-basepython =
-    python3.4
-setenv =
-    PURE_PYTHON = 1
-    PIP_CACHE_DIR = {envdir}/.cache
```

Comment:
This is the sort of thing that could very nicely be handled using [tox's factors](https://tox.readthedocs.io/en/latest/config.html#factors-and-factor-conditional-setting) without multiple [testenv:...] sections.

```ini
[testenv]
setenv =
    ZOPE_INTERFACE_STRICT_IRO=1
    pure: PURE_PYTHON=1
    pure: PIP_CACHE_DIR={envdir}/.cache
```

Reply:
Thanks for the link. I've seen that in one other package but hadn't bothered to find the docs yet.